letsencrypt/plugins/manual.py

```diff
@@ -37,7 +37,7 @@ class ManualAuthenticator(common.Plugin):
 Make sure your web server displays the following content at
```

Comment:
(I know this is irrelevant to this particular PR) This can probably be cleaned up to make it more clear.

In order to prove ownership of the webserver.  Please place this file on the target webserver at .well-known....

If you don't have a webserver currently installed.... run....